frontend/lib/store.ts

```diff
@@ -0,0 +1,34 @@
+import { clubsApi } from "./api/club";
+import { eventsApi } from "./api/event";
+import { usersApi } from "./api/user";
+import { categoriesApi } from "./api/category";
+import { contactsApi } from "./api/contact";
+import { tagsApi } from "./api/tag";
+
+import { configureStore } from "@reduxjs/toolkit";
+import { setupListeners } from "@reduxjs/toolkit/query";
+
+export const store = configureStore({
+  reducer: {
+    [usersApi.reducerPath]: usersApi.reducer,
+    [clubsApi.reducerPath]: clubsApi.reducer,
+    [eventsApi.reducerPath]: eventsApi.reducer,
+    [categoriesApi.reducerPath]: categoriesApi.reducer,
+    [contactsApi.reducerPath]: contactsApi.reducer,
+    [tagsApi.reducerPath]: tagsApi.reducer,
```

Comment:
for example, if I want a custom reducer for the dashboard it would become available to all repos